packages/flutter_hooks/test/use_tab_controller_test.dart

```diff
@@ -138,6 +138,27 @@ void main() {
       verifyNoMoreInteractions(vsync);
       ticker.dispose();
     });
+    testWidgets('initial animationDuration matches with real constructor', (tester) async {
+      late TabController controller;
+      late TabController controller2;
+
+      final vsync = TickerProviderMock();
+      final ticker = Ticker((_) {});
+      when(vsync.createTicker((_) {})).thenReturn(ticker);
```

Comment:
No need to mock tickers. We can use `useSingleTickerProvider()`

Reply:
remove mocked tickers and now use `useSingleTickerProvider()`